app/src/main/java/com/android/sample/model/ingredient/OpenFoodFactsIngredientRepository.kt

```diff
@@ -152,4 +153,23 @@ class OpenFoodFactsIngredientRepository(private val client: OkHttpClient) : Ingr
               }
             })
   }
+
```

Comment:
This is a well-written function to check if the name is available in different languages. In the future, we could even enhance this with a language feature, for example, retrieving the product name in the user's preferred language. For now, prioritizing `""` and `"en"` as the order is a great approach!

Could you please add some comments explaining the parameters and return value of the method for clarity? Thank you!